src/svg.resize.js

```diff
@@ -111,10 +114,42 @@ class ResizeHandler {
     this.el.move(box.x, box.y).size(box.width, box.height)
   }
 
+  rotate (e) {
+    const endPoint = this.el.point(getCoordsFromEvent(e))
+    const cx = this.el.cx()
+    const cy = this.el.cy()
```

Comment:
you already have this knowledge saved in this.box.cx and this.box.cy. You dont want to call bbox TWICE on every mousemove (cx and cy both call bbox)